src/simple_pathfinding.cpp

```diff
@@ -255,16 +284,123 @@ direction reverse_direction( direction dir )
     return direction_from( -direction_to_tripoint( dir ) );
 }
 
-int adjust_omt_cost( int base_cost, direction dir_in, direction dir_out )
+// rounded
+// NOLINTNEXTLINE(cata-large-inline-function)
```

Comment:
This check forbids inline functions in .cpp files. I want these functions to be inlined. They only exist to make the math in the calculations below easier to read consistently. I don't want to put them in the header because other files that include the header don't need to know about them.